features/alias.feature

```diff
@@ -15,9 +15,11 @@ Feature: hub alias
     When I successfully run `hub alias`
     Then the output should contain exactly:
       """
-      # Wrap git automatically by adding the following to ~/.config/fish/config.fish:
+      # Wrap git automatically by adding the following to ~/.config/fish/features/git.fish:
 
-      eval (hub alias -s)\n
+function git --description 'Alias for hub, which wraps git to provide extra functionality with GitHub.'
+	hub $argv
```

Comment:
This line in Go is now indented with a tab character, while the same line here in the test is indented using spaces, so the test fails because of this mismatch. It's not fault, because it was probably hard to predict that this is going to happen, but it makes sense since we indent Go with tabs and Cucumber stories (`*.features`) with spaces.

Maybe try replacing this indentation with a `\t` string here?